locales/pt.yml

```diff
@@ -37,15 +37,16 @@ pt:
     arabic: Árabe
     aragonese: Aragonês
     armenian: Arménio
+    artificial: Artificial (outro)
     australian: Línguas australianas
     basque: Basco
     bengali: Bengali
     bosnian: Bósnio
     bulgarian: Búlgaro
     catalan: Catalão
-    celtic_other: Celtic (Other)
+    celtic_other: Celtic (outro)
     chinese: Chinês
-    creoles_pidgins_other: Creoles and Pidgins (Other)
+    creoles_pidgins_other: Creoles and Pidgins (outro)
```

Comment:
Same here... keep "and"?

Reply:
I don't have the official PT text.

Reply:
I did not notice the languages were in English still, I can go back to "other" but then we have some that say "outro" and others that say "other"

Reply:
Maybe best have your formulation with English and "Translation missing" so we can find it later and ask.

Reply:
It was more just to point out that this still existed. I am happy to merge if this is OK, but thought I would point it out in case it had simply been overlooked.

Reply:
And this one too